cloudpickle/cloudpickle.py

```diff
@@ -93,6 +94,11 @@
     PY3 = True
     PY2 = False
 
+    if platform.python_implementation() == 'PyPy':
+        from importlib._bootstrap import _find_spec
+    else:
+        from _frozen_importlib import _find_spec
```

Comment:
I don't know if other Python implementation behave and we don't support them officially, but maybe it would be nice to do something like:

```python
try:
    from _frozen_importlib import _find_spec
except ImportError:
    # required for PyPy compat
    from importlib._bootstrap import _find_spec
```

Reply:
Yes I agree.

Reply:
Actually, I just realized both ways to import `_find_spec` work for both implementations. This was pure noise.